fastlane/lib/helpers.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+ASC_API_KEY_ENV_VARS = %w[
+  APP_STORE_CONNECT_API_KEY_KEY_ID
+  APP_STORE_CONNECT_API_KEY_ISSUER_ID
+  APP_STORE_CONNECT_API_KEY_KEY
+].freeze
+
+# Fastlane match also uses MATCH_PASSWORD, but we don't expect it to be set because when running locally folks my want to use the password-in-keychain workflow.
```

Comment:
```suggestion
# Fastlane match also uses MATCH_PASSWORD, but we don't expect it to be set because when running locally folks may want to use the password-in-keychain workflow.
```